### ISSUE TITLE
build time errors 

### ISSUE THREAD

**Issue by brainjuic** [arihant-jain-09/discord-clone#4]
there is build time dependencies errors, i have tried my best but in vain.

**Comment by DeveloperParth** [comment]
What are the errors? It'd be nice to be able to see the errors

**Comment by brainjuic** [comment]
![image](https://user-images.githubusercontent.com/118450472/203801757-3636da22-339f-4594-b03b-08a376468b25.png)

**Comment by brainjuic** [comment]
![image](https://user-images.githubusercontent.com/118450472/203801908-9353f132-59d1-408a-8b26-243e9590bf5a.png)

**Comment by brainjuic** [comment]
at initial stages there errors were
[Error message "error:0308010C:digital envelope routines::unsupported"](https://stackoverflow.com/questions/69692842/error-message-error0308010cdigital-envelope-routinesunsupported)

**Comment by brainjuic** [comment]
i had tried this
https://www.freecodecamp.org/news/error-error-0308010c-digital-envelope-routines-unsupported-node-error-solved/

**Comment by brainjuic** [comment]
![image](https://user-images.githubusercontent.com/118450472/203914410-881ba8e6-d6a7-48f3-8558-9f0d65e5d558.png)

**Comment by DeveloperParth** [comment]
So my laptop is being repaired so I can't really replicate it and try to solve it (Vercel) however I've faced similar error in Netlify a few weeks ago and solved it by changing the build command to `CI= npm run build` (maybe it might work for Vercel as well)

**Comment by brainjuic** [comment]
CI= npm run build dependencies remain same?

**Comment by brainjuic** [comment]
![image](https://user-images.githubusercontent.com/118450472/203922476-0b225dc6-fd45-4a13-8244-e6bb7b3a66a4.png)
 tried some changes on vercel and it deployed but with no view.
i have tried deployment of this site on "vercel", "netlify","cpanel", "gihubpages", "firebase". at some platform it show errors and at some platforms it deployed but there were no view. 
could we catch up on anydesk?

**Comment by DeveloperParth** [comment]
[Netlify issue](https://stackoverflow.com/questions/64468843/netlify-deployment-failed-during-stage-building-site-build-script-returned-n) 

I don't think any desk would be possible as I'm at college right now.
Also if you are facing errors with all of those providers I think it might be a project bug have you tried buliding it on your machine? Also try the Netlify solution it might work

**Comment by brainjuic** [comment]
yes i have tried building the project on my machine and deploy it on cpanel but no view.

**Comment by brainjuic** [comment]
![image](https://user-images.githubusercontent.com/118450472/203925714-d197ff8e-1501-4d62-9d55-7382429dedc8.png)

after replacing CI= npm run build, not got the solution

**Comment by brainjuic** [comment]
on my local machine it is running perfectly fine.
![image](https://user-images.githubusercontent.com/118450472/203926377-f42a0a5a-db8c-4d1e-aefa-94a0ce788a13.png)

**Comment by brainjuic** [comment]
please let me know when you will be available ? for to meet up?

**Comment by brainjuic** [comment]
![image](https://user-images.githubusercontent.com/118450472/203932134-c3513a1e-537c-4711-bf02-ee1df9cf51d3.png)
netlify
![image](https://user-images.githubusercontent.com/118450472/203932740-dda8b4e6-14d8-4b54-9515-a91c6902cf8d.png)
same as on vercel.

**Comment by brainjuic** [comment]
?

**Comment by Lewipc** [comment]
can i have help too
